### DISCUSSION TITLE
Is there a way to filter "sorted" and "unsorted" assets

### DISCUSSION THREAD

**Discussion by justin13888** [immich-app/immich#9570] [Feature Request]
### The bug

I have a habit of sorting all the photos and videos I have and would appreciate some way to have a view of assets I still need to organize (e.g. into albums, delete).

Photoprism already has something like that:
![image](https://github.com/immich-app/immich/assets/20733699/72e3201b-895d-4ed0-9a72-9954fdd4c113)

### The OS that Immich Server is running on

Ubuntu 22.04

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

v1.105.0 build 139

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
N/A
```

**Comment by mmomjian** [comment]
Thank you. This is a Duplicate of #1621
